src/views/Item/ItemWrapper.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <div>
+    <div class="tabs" role="tablist">
+      <router-link
+        v-for="(tab, index) in tabs"
+        :key="index"
+        :to="{ name: tab.route }"
+        tag="button"
+        class="tabs__tab"
+        role="tab"
+      >
+        {{ tab.label }}
+      </router-link>
+    </div>
+    <router-view v-show="!loading"></router-view>
+  </div>
+</template>
+
+<script>
+import { mapState } from 'vuex';
+import { itemHome as routerGuard } from '@/router/router-guards';
+
+export default {
+  name: 'ItemWrapper',
+
+  async beforeRouteUpdate(to, from, next) {
+    this.loading = true;
+    if (to.params.slug !== from.params.slug) {
+      await routerGuard(to, from, next);
+      this.loading = false;
+    } else {
+      this.loading = false;
+      next();
+    }
+  },
+
+  data: () => ({
+    loading: true,
+  }),
+
+  computed: {
+    ...mapState(['activeItem']),
+    tabs() {
+      return [
+        {
+          route: 'ItemHome',
+          label: this.$t('general.OKRsLong'),
+        },
+        {
+          route: 'ItemMeasurements',
+          label: this.$t('general.KPIs'),
+        },
+        {
+          route: 'ItemAbout',
+          label: `${this.$t('about.about')} ${this.activeItem.name}`,
+        },
+      ];
+    },
+  },
+
+  mounted() {
+    this.loading = false;
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+.tabs {
```

Comment:
Kunne vi ha samkjørt denne med `TabList`-komponenten som brukes for grafene nå eller på sikt? Nå er de jo ganske like, men oppfører seg bittelitt annerledes.

Reply:
Har lyst til å se på om vi skal videreutvikle `Tab*`-komponenter (ev. se på noe for Punkt?) når vi har landet bruken og utseendet for disse i både (1) navigasjon og (2) innhold. Er litt ulikt i dag, og ikke sikkert f.eks. navigasjon skal beholdes. Tenkte dette var en grei mellomstasjon nå.